autoprotocol/instruction.py

```diff
@@ -17,14 +17,21 @@ class Instruction(object):
 
     builders = InstructionBuilders()
 
-    def __init__(self, op, data):
+    def __init__(self, op, data, informatics=None):
         super(Instruction, self).__init__()
+        if informatics is None:
+            informatics = []
```

Comment:
nit: default informatics directly to `[]` instead of None? Should be equivalent here.

Reply:
Right. I changed this to `[]` then got warning saying `[]` is mutable. Read [this](https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) and changed it back to `None`...

Reply:
Ah, yup. That's right..